src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml.cs

```diff
@@ -56,24 +50,22 @@ public enum CursorState
             ResizeHorizontal
         }
 
-        private Controls.DragCanvas workBench;
+        private DragCanvas workBench;
         private readonly DataTemplate draggedSelectionTemplate;
         private DraggedAdorner draggedAdorner;
         private object draggedData;
         private Point startMousePosition;
         private Point initialMousePosition;
         private PortViewModel snappedPort;
+        private double currentNodeCascadeOffset;
+        private Point inCanvasSearchPosition;
         private List<DependencyObject> hitResultsList = new List<DependencyObject>();
-        Dictionary<CursorState, String> cursorSet = new Dictionary<CursorState, string>();
 
         public WorkspaceViewModel ViewModel
         {
             get
             {
-                if (this.DataContext is WorkspaceViewModel)
-                    return this.DataContext as WorkspaceViewModel;
-                else
-                    return null;
+                return DataContext as WorkspaceViewModel;
```

Comment:
if DataContext is not WorkspaceViewModel it will return null, so there is no need in this if-else